### PR TITLE
Tests failed without reshape paramter due to uninitilized value

### PR DESCRIPTION
**Pull request by SeptimiuIoachimNeagaIntel**
### Details:
Fixed problem in protopipe 
 - if the "reshape" parameter was not set and not needed application would fail with bad variant access

### CODE REVIEW

File: src/plugins/intel_npu/tools/protopipe/src/simulation/simulation.cpp [hunk 1]
```diff
@@ -22,6 +22,14 @@ static cv::gapi::GNetPackage getNetPackage(const std::string& tag, const OpenVIN
         const auto& blob_path = std::get<OpenVINOParams::BlobPath>(params.path);
         network = std::make_unique<P>(tag, blob_path.blob, params.device);
     }
+    if (std::holds_alternative<AttrMap<std::vector<size_t>>>(params.reshape)) {
```
Comment (smirnov-alexey) [inline src/plugins/intel_npu/tools/protopipe/src/simulation/simulation.cpp:25]: Please replicate changes from https://github.com/openvinotoolkit/openvino/pull/27775

**Review by smirnov-alexey — CHANGES_REQUESTED** [review]
Please replicate https://github.com/openvinotoolkit/openvino/pull/27775

File: src/plugins/intel_npu/tools/protopipe/src/simulation/simulation.cpp [hunk 2]
```diff
@@ -22,6 +22,14 @@ static cv::gapi::GNetPackage getNetPackage(const std::string& tag, const OpenVIN
...
+        network->cfgReshape(std::get<AttrMap<std::vector<size_t>>>(params.reshape));
+    } else if (std::holds_alternative<std::vector<size_t>>(params.reshape)) {
+        network->cfgReshape(std::get<std::vector<size_t>>(params.reshape));
+    } else {
```
Comment (AsyaPronina) [inline src/plugins/intel_npu/tools/protopipe/src/simulation/simulation.cpp:29]: Please remove this else when you will be comfortable with removing of debug logs

**Review by smirnov-alexey — APPROVED** [review]

**Review by AsyaPronina — APPROVED** [review]

**Comment by AsyaPronina** [comment]
build_jenkins

**Pull request merged**